pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -202,6 +194,7 @@ def load(fnames, tag=None, inst_id=None, malformed_index=False,
 
     # create very limited metadata
     meta = pysat.Meta()
+    meta[epoch_name] = {'long_name': 'Datetime Index'}
     meta['uts'] = {'units': 's', 'long_name': 'Universal Time'}
     meta['mlt'] = {'units': 'hours', 'long_name': 'Magnetic Local Time'}
```

Comment:
pysat_testing.py has more complete versions of metadata. Perhaps the core variable metadata should be moved into a function in testing.py and called from the other instruments?